frontend/app/routes/$lang+/_public+/apply+/$id+/adult/review-information.tsx

```diff
@@ -495,6 +520,15 @@ export default function ReviewInformation() {
           </ButtonLink>
         </fetcher.Form>
       </div>
+      {useFeature('view-playload') && (
+        <div className="mt-8">
+          <h1 className="text-x1 mb-4 font-semibold">JSON Data</h1>
+          <pre id="playload" className="rounded-log block max-h-96 overflow-auto bg-gray-100 p-4 shadow-inner">
+            {JSON.stringify(playload, null, 2)}
+          </pre>
+          <Button onClick={copyPlayLoadHandler}>copy</Button>
+        </div>
+      )}
```

Comment:
This should be a reusable component that we can drop onto any page.. something like this:

``` tsx
<Debug enableCopy data={payload} />
```

Where `enableCopy` enables/disables the copy button, and `payload` is the raw data that you want to display (the component would `JSON.stringify()` it). This component should also render conditionally, depending on the feature flag.

Something like this:

``` tsx
function Debug({ data, enableCopy = true }) {
  if (featureFlagEnabled) {
    const json = JSON.stringify(data, null, 2);
    const onCopyClick = navigator.clipboard.writeText(json);
    return (
      <>
        <pre>...</pre>
        {enableCopy && <Button>...</Button>}
      </>
    );
  }
}
```

Reply:
new component debug-payload